be/src/common/config.cpp

```diff
@@ -1212,6 +1211,15 @@ DEFINE_mBool(enable_injection_point, "false");
 
 DEFINE_mBool(ignore_schema_change_check, "false");
 
+// The min thread num for BufferedReaderPrefetchThreadPool
+DEFINE_Int64(buffered_reader_prefetch_thread_pool_min_thread_num, "16");
```

Comment:
num_xxx is the native way to describe "how many"
xxx_num is something like the index of something, like nth element of an array